### PR TITLE
Ltd 4708 parallelise e2e

### PR DESCRIPTION
**Pull request by kevincarrogan**
### Aim

Parallelise end-to-end tests to speed up the our overall test runs.

[LTD-4708](https://uktrade.atlassian.net/browse/LTD-4708)

[LTD-4708]: https://uktrade.atlassian.net/browse/LTD-4708?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -254,8 +270,17 @@ jobs:
...
     resource_class: large
+    parallelism: 10
+    environment:
+      PIPENV_YES: 1
     steps:
+      - browser-tools/install-chrome
+      - browser-tools/install-chromedriver
+      - run:
+            name: "Install dependencies"
+            command: pip3 install pipenv
```
Comment (currycoder) [inline .circleci/config.yml:283]: It seems like this string of steps is repeated, could they instead be put inside `prepare_e2e_tests` or some other single setup step?

File: Pipfile [hunk 2]
```diff
@@ -32,13 +32,14 @@ python-dotenv = "*"
...
 django-requests-debug-toolbar = "*"
-moto = {version = "*", extras = ["s3"]}
+moto = {version = "~=4.2", extras = ["s3"]}
 requirements-detector = "==1.2.2"
 pylint = "==2.17.7"
 pylint-celery = "==0.3"
 pylint-django = "==2.5.3"
 pylint-flask = "==0.6"
 pylint-plugin-utils = "==0.7"
+pytest-circleci-parallelized = "*"
```
Comment (currycoder) [inline Pipfile:42]: Should this be more locked down?

**Review by currycoder — APPROVED** [review]
Looks great, just some minor questions

File: Pipfile (hunk 2 above)
Comment (kevincarrogan) [reply Pipfile:42]: For the dev ones I tend not to mind so much, in fact I think it's a bit mad that some of them are very precisely locked down to a single specific version.

File: .circleci/config.yml (hunk 1 above)
Comment (kevincarrogan) [reply .circleci/config.yml:283]: Yes. I can do that.

It wasn't possible for a long time whilst I was developing and I hadn't noticed I'd removed what was blocking me from doing it.

**Pull request merged**